lerna.json

```
@@ -1,7 +1,8 @@
 {
   "packages": [
     "packages/ckb",
-    "packages/ava-site"
+    "packages/ava-site",
+    "packages/lite-insight"
```

```suggestion
    "packages/insight"
```
have to be same with your folder